geonode/client/hooksets.py

```diff
@@ -73,9 +73,6 @@ def map_download_template(self, context=None):
 
     # Map Persisting
     def viewer_json(self, conf, context=None):
-        if not context:
-            context = {}
-
         if isinstance(conf, string_types):
             conf = json.loads(conf)
         return conf

```

Comment:
Not sure of the consequences of returning `None` instead of `{}`.

I would revert this change, maybe by writing it a bit better.

Reply:
It does not change the behavior of the function. `context ` has no effect.